www/include/monitoring/downtime/template/AddDowntime.ihtml

```diff
@@ -154,5 +154,14 @@
 
     initDatepicker("datepicker", "mm/dd/yy", "0");
 
+    $(document).ready(function() {
+        $(".datepicker").first().on('change', function (e) {
+            updateEndTime();
+        });
+        $(".datepicker").last().on('change', function (e) {
+            updateStartTime();
+        });
+    });
```

Comment:
make an update of date with the timepicker too

Reply:
It is not described in the issue